app/src/main/java/com/github/globant/githubsubscribers/subscriberslist/view/SubscribersListView.java

```diff
@@ -12,8 +12,6 @@
  */
 public interface SubscribersListView {
 
-    void navigateToUserDetail();
```

Comment:
This was removed because the fragment is implementing the click listeners so this method is private into the fragment